script/tracker_node.py

```diff
@@ -40,6 +40,12 @@ def image_callback(self, msg):
         header = msg.header
         encoding = msg.encoding
         numpy_image = ros_numpy.numpify(msg)
+
+        # Convert RGBA to RGB
+        if numpy_image.shape[2] == 4:  # Check if the image has 4 channels
```

Comment:
This section assumes that all 4-channel images are RGBA. If the original encoding is `bgra8` or `bgra16`, this could lead to bugs.